lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -78,6 +107,11 @@ export class RefreshTokenClient extends BaseClient {
         return this.acquireToken(refreshTokenRequest);
     }
 
+    /**
+     *
```

Comment:
Was there going to be a comment here?

Reply:
yes. Missed adding, will add now. Thanks for the catch.

Reply:
Done